lib/hermit-remote-add

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+#
+# Summary: Add git remote for hermit profile
+# Usage: hermit remote-add [remote name] <remote address>
+#
+# This simply sets the current profiles git remote to the 
+# address specified. Remote is named origin by default unless otherwise 
+# specified.
+#
+# Copyright 2015, Aaron Griffin
+
+# This file is part of Hermit.
+
+# Hermit is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# Hermit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with Hermit. If not, see <http://www.gnu.org/licenses/>.
+
+set -e
+[ -n "$HERMIT_DEBUG" ] && set -x
+
```

Comment:
We also need to do a check for if this is being called for completions.  You don't need to write the code that will give the proper completions, but you do need to exit early if that's the case.  Check out `hermit use` for what that code looks like.